### DISCUSSION TITLE
本地md文件能显示在线图片，但是部署怎么就不能了？

### DISCUSSION THREAD

**Discussion by elarbor** [hexojs/hexo#5451] [Question / Answer] [answered]
# 本地：
![image](https://github.com/hexojs/hexo/assets/43592730/3a2c72cc-e3e4-46b7-aab6-92bd838118be)

# 部署后：
![image](https://github.com/hexojs/hexo/assets/43592730/5ab0aef5-442b-4cbc-8b58-d1856e857d1d)

https://img-blog.csdnimg.cn/2020021113385684.png?x-oss-process=image/watermark,type_ZmFuZ3poZW5naGVpdGk,shadow_10,text_aHR0cHM6Ly9ibG9nLmNzZG4ubmV0L3FxXzQyNjE4NTY2,size_16,color_FFFFFF,t_70 其实可以单独打开

![image](https://github.com/hexojs/hexo/assets/43592730/9b451764-dcaf-4da7-81fa-08e401d571ca)

**Comment by stevenjoezhang** [comment]
猜测是有防盗链，部署之后请求里面带上 referrer 了会被屏蔽。

**Reply by elarbor** [reply]
![image](https://github.com/hexojs/hexo/assets/43592730/72c85bda-fea8-471a-9283-46baf749124b)

**Reply by stevenjoezhang** [reply]
<img width="1512" alt="截屏2024-04-02 19 21 54" src="https://github.com/hexojs/hexo/assets/16272760/155943d3-5673-4566-bf60-306f2ee6b0fe">
如果你会curl的话，实验一下就知道了

**Reply by elarbor** [reply]
那有解决办法吗？免费的是不只能放在项目里啊？

**Reply by stevenjoezhang** [reply, marked as answer]
可以参考这个 https://cloud.tencent.com/developer/article/1952245

**Reply by elarbor** [reply]
解决了，给每个md文件都加了`<meta name="referrer" content="no-referrer" />`。你说有没有全局只加一次的地方啊😂
目前有111个文件，都加了一遍
![image](https://github.com/hexojs/hexo/assets/43592730/f946fdc9-77fa-4628-86e8-d7e68080ff7d)


![image](https://github.com/hexojs/hexo/assets/43592730/2741aac7-5bb5-4d92-a001-0d0c8da3b2db)
